main.tf

```diff
@@ -0,0 +1,300 @@
+# Terraform
+#--------------------------------------------------------------
+terraform {
+  required_version = ">= 0.10.7"
+}
+
+# Data
+#--------------------------------------------------------------
+data "aws_ssm_parameter" "atlantis_gh_token" {
+  count = "${length(var.github_oauth_token) > 0 ? 0 : 1}"
+  name  = "${local.github_oauth_token_ssm_name}"
+}
+
+data "aws_kms_key" "chamber_kms_key" {
+  count  = "${length(var.kms_key_id) > 0 ? 0 : 1}"
+  key_id = "${local.kms_key_id}"
+}
+
+# Locals
+#--------------------------------------------------------------
+locals {
+  atlantis_gh_webhook_secret  = "${length(var.atlantis_gh_webhook_secret) > 0 ? var.atlantis_gh_webhook_secret : join("", random_string.atlantis_gh_webhook_secret.*.result)}"
+  atlantis_url                = "${format(var.atlantis_webhook_format, local.hostname)}"
+  attributes                  = "${concat(list(var.short_name), var.attributes)}"
+  default_hostname            = "${var.short_name}.${var.domain_name}"
+  github_oauth_token          = "${length(join("", data.aws_ssm_parameter.atlantis_gh_token.*.value)) > 0 ? join("", data.aws_ssm_parameter.atlantis_gh_token.*.value) : var.github_oauth_token}"
+  github_oauth_token_ssm_name = "${length(var.github_oauth_token_ssm_name) > 0 ? var.github_oauth_token_ssm_name : format(var.chamber_format, var.chamber_service, "atlantis_gh_token")}"
+  hostname                    = "${length(var.hostname) > 0 ? var.hostname : local.default_hostname}"
+  kms_key_id                  = "${length(var.kms_key_id) > 0 ? var.kms_key_id : format("alias/%s-%s-chamber", var.namespace, var.stage)}"
+}
+
+# Modules
+#--------------------------------------------------------------
+module "atlantis_ssh_key_pair" {
+  source               = "git::https://github.com/cloudposse/terraform-aws-ssm-tls-ssh-key-pair.git?ref=tags/0.2.0"
+  enabled              = "${var.atlantis_enabled}"
+  namespace            = "${var.namespace}"
+  stage                = "${var.stage}"
+  name                 = "${var.name}"
+  attributes           = "${local.attributes}"
+  ssh_private_key_name = "${var.ssh_private_key_name}"
+  ssh_public_key_name  = "${var.ssh_public_key_name}"
+  ssm_path_prefix      = "${var.chamber_service}"
+}
+
+module "hostname" {
+  source           = "git::https://github.com/cloudposse/terraform-aws-route53-alias.git?ref=tags/0.2.7"
+  enabled          = "${var.atlantis_enabled}"
+  aliases          = ["${local.hostname}"]
+  parent_zone_name = "${var.domain_name}"
+  target_dns_name  = "${var.alb_dns_name}"
+  target_zone_id   = "${var.alb_zone_id}"
+}
+
+module "atlantis_webhooks" {
+  source              = "git::https://github.com/cloudposse/terraform-github-repository-webhooks.git?ref=tags/0.1.1"
+  github_token        = "${local.github_oauth_token}"
+  webhook_secret      = "${local.atlantis_gh_webhook_secret}"
+  webhook_url         = "${local.atlantis_url}"
+  active              = "${var.atlantis_enabled}"
```

Comment:
add `enabled` as well.

```
enabled              = "${var.atlantis_enabled}"
```

or as @osterman suggested, use `var.enabled`
```
enabled              = "${var.enabled}"
```

If `var.atlantis_enabled == false` and without setting `enabled`, `module "atlantis_webhooks"` will be created anyway (the webhook will be not active)